lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/evaluator/EvaluatorManager.java

```diff
@@ -111,17 +108,19 @@
 
   // Mutable fields
   private Optional<TaskRepresenter> task = Optional.empty();
-  private boolean isResourceReleased = false;
-  private boolean allocationFired = false;
+  private boolean resourceNotReleased = true;
+  private boolean allocationNotFired = true;
 
   @Inject
   private EvaluatorManager(
+      @Parameter(EvaluatorIdentifier.class) final String evaluatorId,
```

Comment:
@jwang98052 I don't think we have it written anywhere, but we should - IMHO having `@Parameter` annotations first and keeping them together adds to the code readability.